modules/gatsby/src/gatsby-config/get-gatsby-config.js

```diff
@@ -309,6 +313,19 @@ module.exports = function getGatsbyConfig(config) {
     ]
   };
 
+  // conditional plug-ins - only added depending on options on config
+
+  if (config.DIR_NAME) {
```

Comment:
w/o this check this generates an error for ocular-gatsby project that don't have a DIR_NAME specified in config. 
we can use this pattern to add plugins that won't break for older projects.

Reply:
Do we want to warn if we don't find key variables in ocular-config?